sdk/rust/src/http/router.rs

```diff
@@ -19,6 +19,18 @@ impl Default for Router {
     }
 }
 
+impl Display for Router {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str("Registered routes:\n")?;
+        for (method, router) in &self.methods_map {
+            router.iter().for_each(|route| {
+                f.write_fmt(format_args!("- {}: {}\n", method, route.0)).unwrap();
```

Comment:
To avoid the `unwrap`, could we use `try_for_each` in the loop?

Reply:
Or instead of using an iterator combinator, you can just use another for loop and use the `?` operator.

Reply:
Thanks for the feedback. Just having two for loops is way easier to read. Makes definitely sense. Thanks for the feedback guys!